updates/next.json

```diff
@@ -13,19 +13,19 @@
       }
     },
     {
-      "version": "32.20200907.1.0",
+      "version": "32.20200907.1.1",
       "metadata": {
         "rollout": {
           "start_percentage": 1.0
         }
       }
     },
     {
-      "version": "32.20200907.1.1",
+      "version": "32.20200923.1.0",
       "metadata": {
         "rollout": {
           "duration_minutes": 1440,
```

Comment:
shall we bump this back up to 2880? We bumped it down for the release earlier this week but we should probably go back to 48H as a standard.

Reply:
Sure will do